src/h2o_discovery/__init__.py

```diff
@@ -108,6 +108,6 @@ def _lookup_and_load(
             " Alternatively, you can create configure your local environment with"
             " the with H2O CLI with and/or use H2OCONFIG environment variable"
             " (see https://docs.h2o.ai/h2o-ai-cloud/developerguide/cli)."
-        )
+        ) from None
```

Comment:
QQ: What is the reason to suppress the exception context here?

Reply:
This change is just addressing linting error.

```
src/h2o_discovery/__init__.py:104:9: B904 Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling
```

https://docs.astral.sh/ruff/rules/raise-without-from-inside-except/

So no extra suppressing as the whole chain is custom and explicitly raised.